### ISSUE TITLE
Translation constraints

### ISSUE THREAD

**Issue by toinouu** [FriendsOfSymfony/FOSCommentBundle#587]
Hi 
Why not use symfony default error messages for constraints?

if i remove [this  line](https://github.com/FriendsOfSymfony/FOSCommentBundle/blob/master/Resources/config/validation.xml#L21)
and [this line](https://github.com/FriendsOfSymfony/FOSCommentBundle/blob/master/Resources/config/validation.xml#L25)

i have the same message translated  but with the correct local

thx

**Comment by toinouu** [comment]
When I try to extend FOSCommentbundle 

```xml
    <class name="FOS\CommentBundle\Model\Comment">
        <property name="body">
            <constraint name="NotBlank">
            </constraint>
            <constraint name="Length">
                <option name="min">3</option>
                <option name="max">10000</option>
            </constraint>
        </property>
    </class>
```
i have 2 errors for body field 
- The message is too short
- Cette chaine est trop courte. Elle doit avoir au minimum 3 caractères. (default msg fr)

**Comment by toinouu** [comment]
Issue resolved with last version of bundle 2.0.9
